### PR TITLE
Init

### PR DESCRIPTION
**Pull request by muhamedcpqd**
This is connected to dojot/dojot#601

### CODE REVIEW

**Comment by muhamedcpqd** [comment]
This is connected to dojot/dojot#602

File: package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  "description": "A library that provides utilities and methods for dojot",
+  "main": "index.js",
+  "scripts": {
+    "test": "mocha tests/*.test.js --exit"
+  },
+  "keywords": [
+    "dojot"
+  ],
+  "author": "Muhamed Avila",
+  "license": "ISC",
```
Comment (giovannicuriel) [inline package.json]: GPL 3.0

File: libs/logger.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
+/*Dojot Logger Library*/
+const winston = require('winston');
+const moment = require('moment');
+const express = require('express');
+const { combine, printf, timestamp, colorize} = winston.format;
+
+/*Setting timestamp*/
+moment.locale('pt-br');
```
Comment (giovannicuriel) [inline libs/logger.js]: Is this really needed?

File: libs/logger.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+            level: 'debug',
+            handleExceptions: true
+        })
+    ],
+    exitOnError: false
+});
+
+/*Route to runtime debug change*/
+const loggerRoute = function(app, filename){
+    app.get('/setLog', (req, res) => {
```
Comment (giovannicuriel) [inline libs/logger.js]: We should create an API description.

Comment (muhamedcpqd) [reply libs/logger.js]: ok!

**Pull request merged**